### ISSUE TITLE
radio@driglu4it - Add option or change code to close submenu automatically [feature request]

### ISSUE THREAD

**Issue by Hilyxx** [linuxmint/cinnamon-spices-applets#4934]
**Notify author of applet**
@jonath92 

**Issue**
when radio is not played,  radio applet channels sub menu stay open permanently. This is must be automatically closed.
This is not really an issue but with many radio channels it is not practical.

![Capture d’écran du 2023-07-10 03-01-42](https://github.com/linuxmint/cinnamon-spices-applets/assets/21991448/09160a3e-0f09-4a3a-8cab-43dec9eb391a)

**Comment by jonath92** [comment]
Yes, that's true. Pull requests to fix it are more than welcome.
